internal/services/azapi_resource_data_source.go

```diff
@@ -188,7 +192,18 @@ func (r *AzapiResourceDataSource) Read(ctx context.Context, request datasource.R
 		id = buildId
 	}
 
-	client := r.ProviderData.ResourceClient
+	var client clients.Requester
+	client = r.ProviderData.ResourceClient
+	if !model.RetryableErrors.IsNull() && !model.RetryableErrors.IsUnknown() {
+		bkof, regexps := clients.NewRetryableErrors(
+			model.RetryableErrors.GetIntervalSeconds(),
```

Comment:
The default value seem doesn't work on a data source. So the inverval second becomes 0 when using the retry feature with a data source.

Reply:
Fixed in cb47241e2e5de914d6f74a3d1beda911dc950db9